src/Parser/Druid/Feral/Modules/Features/FerociousBiteEnergy.js

```diff
@@ -0,0 +1,90 @@
+import React from 'react';
+
+import Analyzer from 'Parser/Core/Analyzer';
+import Combatants from 'Parser/Core/Modules/Combatants';
+
+import SPELLS from 'common/SPELLS';
+import SpellLink from 'common/SpellLink';
+import Wrapper from 'common/Wrapper';
+
+const energyResourceType = 3;
+const energyForFullDamage = 50;
+
+/**
+ * Although Ferocious Bite costs 25 energy, it does up to double damage if the character has more.
+ * It's recommended that feral druids use Bite when at 50 energy or higher.
+ * An exception to this is when the bonus from 4-piece T21 is active, which makes Bite cost no
+ * energy and ignore the current energy level.
+ */
+class FerociousBiteEnergy extends Analyzer {
+  static dependencies = {
+    combatants: Combatants,
+  };
+  
+  lowEnergyBites = 0;
+  sumEnergyBelowTarget = 0;
+
+  on_byPlayer_cast(event) {
+    if (event.ability.guid !== SPELLS.FEROCIOUS_BITE.id) {
+      return;
+    }
+
+    if (event.classResources[0].type !== energyResourceType) {
+      // classResources[0] not being energy means Ferocious Bite didn't consume any energy,
+      // and so ignores the character's energy level.
+      return;
+    }
+    if (event.classResources[0].type === energyResourceType && !event.classResources[0].cost) {
+      // Ferocious Bite didn't consume energy, and so ignores the character's energy level.
+      return;
+    }
+
+    // 'amount' is the resource present before the spell alters it.
+    const energy = event.classResources[0].amount;
+    if (energy < energyForFullDamage) {
+      this.lowEnergyBites++;
+      this.sumEnergyBelowTarget += (energyForFullDamage - energy);
+    }
+  }
+
+  /**
+   * Use a sum of the magnitude of error to trigger suggestion rather than a simple count
+   * of low energy bites. This way using Bite at 49 energy is correctly seen as a minor error
+   * with only slight damage loss, while a Bite at 25 energy is more significant.
+   */
+  get suggestionThresholds() {
+    return {
+      actual: this.sumEnergyBelowTarget,
+      isGreaterThan: {
+        minor: 5,
+        average: 25,
+        major: 60,
+      },
+      style: 'number',
+    };
+  }
+
+  suggestions(when) {
+    when(this.suggestionThresholds).addSuggestion((suggest, actual, recommended) => {
+      let actualText = '';
```

Comment:
I would suggest that you move this text to the suggest text instead (Just add it infront of your current suggest text) and then use the actual variable you get from the threshold as the actual text. That way your specific text on how many bites that was messed up and by how much is more in focus.